.npmrc

```diff
@@ -0,0 +1,3 @@
+registry=https://registry.npmjs.org/
+@navikt:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=${NPM_AUTH_TOKEN}
```

Comment:
Dersom vi skal endre til å bruke token for å kjøre npm i burde det skrives i README hvordan opprette og sette token lokalt. Er tanken å sette path variabel? Legge det direkte inn i npmrc?

Reply:
Oppdaterte README med en liten beskrivelse på hva som må gjøre